lib/cookie.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+
+ // needed to communicate/translate between JSONWire cookies and regular JS cookies 
+
+import _ from 'lodash';
+import { getLogger } from 'appium-logger';
+
+
+export default class Cookie {
+
+  constructor (cookieString) {
+    this.cookieString = cookieString;
+    this.log = getLogger('mymodule');
```

Comment:
Need a better name than "mymodule". This text will be prefixed before each log line generated in this module. Perhaps "Cookie" would be a better option?